plugins/solaris/fsstat_act_

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
```

Comment:
Here you are using only one bash feature: `${...^^}` (uppercase conversion).
Maybe you would like to switch to `/bin/sh` and use `tr '[a-z]' '[A-Z]'` for case conversion?